hail/python/test/hailtop/batch/test_regenie.py

```diff
@@ -0,0 +1,83 @@
+import unittest
+import hailtop.batch.genetics.regenie as br
+import os
+from shutil import which, rmtree
+
+
+rdir = "hailtop/batch/genetics/regenie"
+
+
+def read(file):
+    with open(file, 'r') as f:
+        return f.read()
+
+
+def assert_same_file(file1, file2):
+    assert read(file1) == read(file2)
+
+
+class LocalTests(unittest.TestCase):
```

Comment:
These tests do not enable continuous integration because they depend on docker. Regenie, due to its use of custom images, cannot use LocalBackend in CI.

Let's remove all the `skipIf`, change the name to BatchTests, and follow the style of test_batch.py's BatchTests (which really should be called ServiceTests).

Reply:
This PR does not have a service backend version. These tests do not block CI, they are just skipped